utils/shadow/patches/005-set-encrypt-method-sha512.patch

```diff
@@ -1,11 +0,0 @@
---- a/etc/login.defs
```

Comment:
I don't like this. ping @Cynerd

Reply:
Note that this patch is replaced by one of the SED lines.